_episodes/01-shell-basics.md

```diff
@@ -258,10 +254,10 @@ unintentionally run a different version and end up with different results.
 {: .callout}
 
 
-> ## How can you add a new path for shell to look for commands in?
+> ## How can you add a new path in which shell will look for commands…
```

Comment:
so `…` because there is more text to the question? might be nice indeed - anywhere else I use such form?

Reply:
I'll keep an eye out!